tests/vector/test-vector.c

```diff
@@ -22,7 +22,7 @@
 
 #include <noise/protocol.h>
 #include "json-reader.h"
-#include <setjmp.h>
+// #include <setjmp.h>
```

Comment:
To me this line shouldn't be commented but rather but in a #define section. It will ease the maintenance

Reply:
Yes, done